StudioCore/TextEditor/TextEditorScreen.cs

```diff
@@ -335,12 +335,12 @@ private void FMGSearchLogic()
                         if (_FMGsearchStr.Length > _FMGsearchStrCache.Length)
                             searchEntries = _cachedEntriesFiltered;
                         else
-                            searchEntries = _cachedEntries;//FMGBank.GetItemFMGEntriesByType(_activeItemCategory, FMGBank.ItemType.Title).ToList()
+                            searchEntries = _cachedEntries;
 
                         foreach (var entry in searchEntries)
                         {
-                            if (entry.ID.ToString().Contains(_FMGsearchStr, StringComparison.CurrentCultureIgnoreCase))
-                                matches.Add(entry);
+                            if (entry.ID.ToString().Contains(_FMGsearchStr))
+                                matches.Add(entry); //ID matches
```

Comment:
We're moving to case sensitive? Also added comment doesn't seem needed and I'd prefer comments to be on their own line almost always.

Reply:
This specifically searches IDs, so case check was unnecessary (though I suppose any impact is beyond negligible). ID match comment was to note that this was for ID searching (I'll move the comment)